pywren/wrenhandler.py

```diff
@@ -318,9 +324,16 @@ def consume_stdout(stdout, queue):
                     response_status[key] = float_value
 
         end_time = time.time()
+        response_status['retcode'] = process.returncode
 
         response_status['stdout'] = stdout.decode("ascii")
 
+        if process.returncode != 0:
```

Comment:
Is it better to do this right after the process finishes (line 317) ? I'm not sure if we want to run through the `JOBRUNNER_STATS_FILENAME` loop or not in this case